server/.env.example

```diff
@@ -19,6 +19,11 @@ SIGN_IN_PREFILLED=true
 # REFRESH_TOKEN_EXPIRES_IN=90d 
 # FRONT_AUTH_CALLBACK_URL=http://localhost:3001/verify 
 # AUTH_GOOGLE_ENABLED=false
+# AUTH_GOOGLE_GMAIL_ENABLED=false
```

Comment:
I think we should change these naming to:
EMAIL_PROVIDER_GMAIL_ENABLED
EMAIL_PROVIDER_GMAIL_CLIENT_ID...

We will also need to add them to the doc

Reply:
Maybe MESSAGING_PROVIDER_... instead, since we will have other messaging systems later, not just emails